src/workloads/scale/MixedWorkloadsGennyStress.yml

```diff
@@ -144,7 +144,7 @@ ActorTemplates:
 Actors:
 - Name: Setup
   Type: Loader
-  Threads: 1
+  Threads: 8
   Phases:
   - Repeat: 1
     BatchSize: 100
```

Comment:
I'm pretty sure the next line does nothing now (and is never even accessed).

Reply:
Yes -- I meant to delete it. Will push that on up. I did not go and remove all the vestigal thread settings though.